core/src/vminterface.rs

```diff
@@ -119,31 +77,21 @@ pub enum AvmObject<'gc> {
 }
 
 impl<'gc> AvmObject<'gc> {
-    /// Determine if this object is an AVM1 object.
-    pub fn is_avm1_object(&self) -> bool {
-        matches!(self, Self::Avm1(_))
-    }
-
     /// Attempt to access the AVM1 claim to this object, generating an error if
```

Comment:
comment not up to date.
(also, are you sure we don't want an error anymore?)

Reply:
Updated comments. I switched over to `Option` because `ClaimError` was equivalent to `()`, and these methods aren't really used as error-returning. Most callers immediately converted to `Option` using `.ok()`, and the others matched against `Ok(...)` and `Err(...)`, without ever propagating errors.